spatialprofilingtoolbox/ondemand/providers/__init__.py

```diff
@@ -0,0 +1,7 @@
+"""Classes that provide metric calculation on demand."""
+
+from spatialprofilingtoolbox.ondemand.providers.provider import Provider
+from spatialprofilingtoolbox.ondemand.providers.pending_provider import PendingProvider
+from spatialprofilingtoolbox.ondemand.providers.counts_provider import CountsProvider
+from spatialprofilingtoolbox.ondemand.providers.proximity_provider import ProximityProvider
+from spatialprofilingtoolbox.ondemand.providers.squidpy_provider import SquidpyProvider
```

Comment:
I systematically failed to consider using the `__init__` files to export symbols up the chain of nested modules.
This could be a way to reduce the verbosity and repetition of the single-symbol-per-import-line style that I have been recommending.